src/pages/Balance.vue

```diff
@@ -142,10 +142,12 @@
     <DepositEVM
       v-model:showDepositEVMDlg="showDepositEVMDlg"
       v-model:nativeTLOSBalance="coins[0].amount"
+      @updateBalances="updateBalances"
```

Comment:
both of these components, `DepositEVM` and `WithdrawEVM` should have `emits` added to their component definition:
```js
export default {
    name: '...',
    emits: ['updateBalances'], // here
    methods: { ... }
}
```

https://vuejs.org/guide/components/events.html#declaring-emitted-events
I haven't checked yet for this application but on other vue3 projects i've touched, vue throws a warning if custom events are not declared